src/shared/cloudformation/templateSymbolResolver.ts

```diff
@@ -113,19 +114,13 @@ export class TemplateSymbolResolver {
 
 export class TemplateSymbolProvider {
     public async getSymbols(document: vscode.TextDocument, waitForSymbols: boolean): Promise<vscode.DocumentSymbol[]> {
-        for (let i = 0; i < 30; i++) {
-            const symbols = await vscode.commands.executeCommand<vscode.DocumentSymbol[]>(
+        const symbols = await waitUntil(async function() {
+            return await vscode.commands.executeCommand<vscode.DocumentSymbol[]>(
```

Comment:
Out of curiosity, in what cases would this not return on first try?

Reply:
It's asynchronous, there is no guarantee that symbols are available. This causes tests to fail if the symbols are not available in time.

Reply:
OOS for this PR, but is it worth porting this to the other `executeDocumentSymbolProvider` calls?